### PR TITLE
[#25] [UI] As a user, I can submit survey's answers

### LINKED ISSUE #25: [UI] As a user, I can submit survey's answers
## Why
* Allow user to submit answer

## Acceptance Criteria

* If submit successfully, show thank you screen
* If fail to submit, show error alert

## Design

Screen Shot 2022-05-09 at 16 18 17

### PR DESCRIPTION
**Pull request by ducbm051291**
- Close #25 

## What happened 👀

- Create LottieDialog
- Add function showLottieDialog in Context extension
- Add Lottie library in pubspec
- Update localizable strings of thanks for taking survey in 3 languages
- Rename widget folder to widgets
- Add submit_successfully.json file for lottie

## Insight 📝

- N/A

## Proof Of Work 📹

CleanShot 2023-05-09 at 08 59 05

### CODE REVIEW

File: lib/ui/widgets/lottie_dialog.dart [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Dialog(
+      backgroundColor: Colors.black,
+      insetPadding: const EdgeInsets.all(0),
+      child: Container(
+        width: MediaQuery.of(context).size.width,
+        height: MediaQuery.of(context).size.height,
```
Comment (nmint8m) [inline lib/ui/widgets/lottie_dialog.dart:43]: Please use `context.screenSize` to shorten the code.

File: lib/ui/widgets/lottie_dialog.dart [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+                _controller.forward().then(
+                      (_) => Future.delayed(const Duration(seconds: 2),
+                          () => widget.onAnimated()),
+                    );
+              },
+            ),
+            const SizedBox(height: 20),
+            Text(
+              AppLocalizations.of(context)?.surveyThanks ?? '',
+              style: Theme.of(context).textTheme.displayMedium,
```
Comment (nmint8m) [inline lib/ui/widgets/lottie_dialog.dart:63]: Please use `context.localization` and `context.textTheme` to shorten the code.

**Review by nmint8m — COMMENTED** [review]
There are a few comments, please check. 😄

**Review by nmint8m — APPROVED** [review]
Look good to me ✅

File: lib/ui/widgets/lottie_dialog.dart [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+  @override
+  Widget build(BuildContext context) {
+    return Dialog(
+      backgroundColor: Colors.black,
+      insetPadding: const EdgeInsets.all(0),
+      child: Container(
+        width: context.screenSize.width,
+        height: context.screenSize.height,
+        padding: const EdgeInsets.all(20),
```
Comment (thiennguyen0196) [inline lib/ui/widgets/lottie_dialog.dart:44]: We can use `paddingMedium` here
```suggestion
        padding: const EdgeInsets.all(Dimensions.paddingMedium),
```

**Review by thiennguyen0196 — COMMENTED** [review]
Just one concern, the rest lgtm 👏

**Pull request merged**